Formula/boost-build.rb

```diff
@@ -1,8 +1,8 @@
 class BoostBuild < Formula
   desc "C++ build system"
   homepage "https://www.boost.org/build/"
-  url "https://github.com/boostorg/build/archive/boost-1.77.0.tar.gz"
-  sha256 "17ad1addbc08d1cc6ef52f7140097915bc4904c28c7d6d733c4a1a20d40bbc1c"
+  url "https://boostorg.jfrog.io/artifactory/main/release/1.78.0/source/boost_1_78_0.tar.bz2"
```

Comment:
This is the full boost source - not the boost-build source.

If you really want to use the full Boost source, then you'll need to cd to `tools/build`.

Reply:
@SMillerDev can we go back to the GitHub archive?